crates/web-plugins/didcomm-messaging/protocols/mediator-coordination/src/plugin.rs

```diff
@@ -0,0 +1,27 @@
+use message_api::{MessagePlugin, MessageRouter, PluginError};
+
+#[derive(Default)]
+pub struct MediatorCoordination;
+
+impl<S, M, R> MessagePlugin<S, M, R> for MediatorCoordination
+where
+    S: Clone + Sync + Send + 'static,
+    M: Send + 'static,
+    R: Send + 'static,
+{
+    fn name(&self) -> &'static str {
+        "mediator-coordination"
+    }
+
+    fn mount(&mut self) -> Result<(), PluginError> {
+        Ok(())
+    }
+
+    fn unmount(&self) -> Result<(), PluginError> {
+        Ok(())
+    }
+
+    fn routes(&self) -> MessageRouter<S, M, R> {
+        MessageRouter::new()
```

Comment:
Indeed, the `routes` function registers all the routes used by a **didcomm-messaging** protocol. You are providing an empty router here.